app/helpers/manage_metadata_helper.rb

```diff
@@ -1,6 +1,6 @@
 # :nodoc:
 module ManageMetadataHelper
-  def display_entry_id(metadata, type)
+  def breadcrumb_name(metadata, type)
```

Comment:
I like the name change, its much clearer what the gem is doing and what MMT is doing.